cvrf2csaf/common/utils.py

```diff
@@ -1,19 +1,24 @@
 import argparse
 import yaml
 import os, logging, json
+import pkg_resources
 from pathlib import Path
 
-def get_config_from_file():
-    """Loads configuration file. Parts of it can be overwritten by CLI arguments."""
+
+def get_config_from_file() -> dict:
+    """ Loads configuration file. Parts of it can be overwritten by CLI arguments. """
     config = dict()
     try:
-        f = open('./config.yaml', 'r')
-        config = yaml.safe_load(f)
+        # TODO: Workaround for now, config file placement is to be discussed
+        req = pkg_resources.Requirement.parse('cvrf2csaf')
+        path_to_conf = pkg_resources.resource_filename(req, 'cvrf2csaf/config/config.yaml')
+        with open(path_to_conf, 'r') as f:
+            config = yaml.safe_load(f)
```

Comment:
Nit: a single line body of `return yaml.safe_load(f)` should be enough though ...

Reply:
For the sake of explicitness, let's leave it like this